tests/foreman/api/test_repository.py

```diff
@@ -303,62 +658,47 @@ def test_positive_create(self):
                     product=product,
                     url=DOCKER_REGISTRY_HUB,
                 ).create()
-                repo2 = repo.read()
-                self.assertEqual(repo.name, repo2.name)
+                self.assertEqual(repo.name, name)
                 self.assertEqual(
-                    repo.docker_upstream_name,
-                    repo2.docker_upstream_name
-                )
-                self.assertEqual(repo.content_type, repo2.content_type)
+                    repo.docker_upstream_name, u'busybox')
+                self.assertEqual(repo.content_type, u'docker')
 
     @tier2
     @run_only_on('sat')
-    @skip_if_bug_open('bugzilla', 1217603)
     def test_positive_synchronize(self):
         """Create and sync a Docker-type repository
 
-        @Assert: A repository is created with a Docker repository
-        and it is synchronized.
+        @Assert: A repository is created with a Docker repository and it is
+        synchronized.
 
         @Feature: Repository
         """
-        from pprint import PrettyPrinter
-        printer = PrettyPrinter().pprint
-
         product = entities.Product(organization=self.org).create()
-        repository = entities.Repository(
+        repo = entities.Repository(
             content_type=u'docker',
             docker_upstream_name=u'busybox',
             name=u'busybox',
             product=product,
             url=DOCKER_REGISTRY_HUB,
         ).create()
-        printer(repository.id)
-        printer(repository.sync())
-        self.assertGreaterEqual(
-            repository.read_json()[u'content_counts'][u'docker_image'],
-            1
-        )
+        repo.sync()
+        self.assertGreaterEqual(repo.read().content_counts['docker_image'], 1)
 
     @tier1
+    @skip_if_bug_open('bugzilla', 1194476)
     def test_positive_update_name(self):
```

Comment:
if this is still `docker`-only, i would reflect that in the name of the test `test_positive_docker_update_name` ..or similar.